source/backend/dal/Repositories/CompensationRequisitionRepository.cs

```diff
@@ -50,7 +50,15 @@ public PimsCompensationRequisition GetById(long compensationRequisitionId)
         {
             User.ThrowIfNotAuthorized(Permissions.CompensationRequisitionView);
             var entity = Context.PimsCompensationRequisitions
+                .Include(x => x.YearlyFinancial)
+                .Include(x => x.ChartOfAccounts)
+                .Include(x => x.Responsibility)
                 .Include(c => c.PimsCompReqH120s)
+                    .ThenInclude(y => y.FinancialActivityCode)
+                .Include(x => x.PimsAcquisitionPayees)
```

Comment:
@FuriousLlama you already have a separate require for getting payees for a comp req right? does that mean we can take this out and better stick to our limitation of nested entities?

Reply:
would rather still keep it for the details view.